Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/docrepository/adapter/AdapterComponentDocRepositoryOrchImplTest.java

```diff
@@ -84,28 +88,25 @@
     @Test
     public void testGetDocumentService() {
         AdapterComponentDocRepositoryOrchImpl docRepo = new AdapterComponentDocRepositoryOrchImpl();
-        assertTrue(docRepo.getDocumentService() instanceof DocumentService);
+        assertTrue(docRepo.getDocumentService() != null);
```

Comment:
This always return true

Reply:
I think this was just one of those Eclipse "On Save" rules, like the tabs to spaces. I didn't directly modify this particular line. Same goes for the other comments regarding this type of change.

I have many problems with this entire test class, not just this particular bit.